llama-index-integrations/embeddings/llama-index-embeddings-huggingface/llama_index/embeddings/huggingface/base.py

```diff
@@ -204,15 +143,11 @@ async def _aget_text_embedding(self, text: str) -> List[float]:
 
     def _get_text_embedding(self, text: str) -> List[float]:
         """Get text embedding."""
-        text = format_text(text, self.model_name, self.text_instruction)
-        return self._embed([text])[0]
+        return self._embed(text, prompt_name="text")
 
     def _get_text_embeddings(self, texts: List[str]) -> List[List[float]]:
         """Get text embeddings."""
-        texts = [
-            format_text(text, self.model_name, self.text_instruction) for text in texts
-        ]
-        return self._embed(texts)
+        return self._embed(texts, batch_size=len(texts), prompt_name="text")
```

Comment:
ah, this is actually set by `self.embed_batch_size`

Reply:
Ah, I see! I thought the batch size was only used in the superclass. I'll update to use `self.embed_batch_size`.

Reply:
Oh wait, yes, somehow you knew better than me already 😆 By the time this is called, it will already be `embed_batch_size` length -- but I guess either way is fine

Reply:
Resolved via 11ae153e72863ffa735d8aa6ec2b3bdb90cf40dc